### PR TITLE
Added features for unclaiming and unassigning issues

### LINKED ISSUE #61: Add un-assign and un-claim commands on github
## Description
As a developer,
I need add an unclaim and unassign commands to sysbots ability on github.

### Update [Required]
- [ ] Add unassign and unclaim commands on github

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1.5 hours

### PR DESCRIPTION
**Pull request by sammy1997**
**Depends on:
   Please merge it only after merging #64**
# Description
Added features for un-claiming and un-assigning issues.

Fixes #61 

# Type of Change:
- Code
- Documentation
- This change requires a documentation update (software upgrade on readme file)
- New feature (a non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?

unclaim_unassign

# Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by sammy1997** [comment]
Please merge it only after merging #64 .

File: github_functions.py [hunk 1]
```diff
@@ -145,3 +150,21 @@ def open_issue_github(repo_owner, repo_name, issue_title, issue_body, author):
...
+
+def get_issue_author(repo_owner, repo_name, issue_number):
+    session = requests.Session()
+    session.auth = (USERNAME, PASSWORD)
+    request_url = get_issue_url % (repo_owner, repo_name, issue_number)
+    response = session.get(request_url).json()
+    return response.get('user', {}).get('login', '')
+
+
+def unassign_issue(repo_owner, repo_name, issue_number, assignee):
```
Comment (sara-02) [inline github_functions.py]:
```python
def unassign_issue(repo_owner, repo_name, issue_number, assignee):
    if not assignee:
       return <some messgae like"No one assigned the issue yet">
    session = requests.Session()
    session.auth = (USERNAME, PASSWORD)
    request_url = remove_assignee_url % (repo_owner, repo_name, issue_number)
    headers = {'Accept': 'application/vnd.github.symmetra-preview+json', 'Content-Type': 'application/json'}
    data = '{"assignees": ["%s"]}' % assignee
    response = session.delete(request_url, data=data, headers=headers)
    return response.status_code
```
WDYT?

**Comment by sammy1997** [comment]
@sara-02 @ramitsawhney27 The assignee is being passed to this function from other functions, so it won't be null but rather the comment makers user ID. So this check isn't valid I think. WDYT?

**Comment by sara-02** [comment]
@sammy1997 Oh, ok. I thought that `assignee` is a list, and wanted a check that if the list is empty i.e no one is assigned the issue yet, then we don't have to call the unassigned function.

**Comment by sammy1997** [comment]
Basically who ever comments @sys-bot unassign/unclaim, this function gets his/her username as assignee

**Comment by sara-02** [comment]
I see 2 cases here.
Case 1: I am a contributor, I do `sysbot claim`, I get assigned the issue. Now I do `sysbot unclaim` or `sysbot unassign` and the issue is no longer assigned to me.
Case 2: I am contributor, I do `sysbot claim`, I get assigned the issue. Now I abandon the issue, and do not reply to any queries. A maintainer calls `sysbot unassign`, and unassigns the issue.
What you are saying will work for case 1. what I proposed was keeping case 2 in mind.

**Comment by sammy1997** [comment]
@sara-02 @sys-bot unassign <username of person to be unassigned> can be used in 2nd case. This function can work in both cases.

**Comment by sammy1997** [comment]
Oh no.....:) For unassigning, just like un assigning, the username of the person to be unassigned will be mentioned in the command( like if I am the maintainer and you are assigned, then I can unassign you if you becoe unresponsive using @sys-bot unassign sara-02 ), while for unclaim only the person who had claimed or assigned this issue can unclaim it( for this command it takes the commenters ID ) format- @sys-bot unclaim

**Comment by sara-02** [comment]
@sammy1997 Got it. Now it is clear to me :)

**Comment by sara-02** [comment]
@ramitsawhney27 you can go through the above discussion once, and then we can approve and merge this.

**Pull request merged**